doc/build-windows.md

```diff
@@ -32,7 +32,7 @@ First, install the general dependencies:
 
     sudo apt update
     sudo apt upgrade
-    sudo apt install cmake curl g++ git make pkg-config
+    sudo apt install cmake curl g++ git make pkg-config xz-utils
```

Comment:
+1, although it's somewhat less convienient, refering to the depends README for what to install would be easier to maintain cleanly. Otherwise, if it changes, people have to think of updating this too.

Reply:
I agree with you, I'm closing the pr.